.travis.yml

```diff
@@ -18,10 +17,10 @@ before_script:
   - cd plugins/flyvemdm && composer install --no-interaction
 
 script:
-  - vendor/bin/atoum --debug -bf tests/bootstrap.php -no-cc --max-children-number 1 -d tests/suite-install
-  - vendor/bin/atoum --debug -bf tests/bootstrap.php -no-cc --max-children-number 1 -d tests/suite-unit
-  - for f in `ls tests/suite-integration/`; do vendor/bin/atoum --debug -bf tests/bootstrap.php -no-cc --max-children-number 1 -f tests/suite-integration/$f; done;
-  - vendor/bin/atoum --debug -bf tests/bootstrap.php -no-cc --max-children-number 1 -d tests/suite-uninstall
+  - vendor/bin/atoum -ft -bf tests/bootstrap.php --max-children-number 1 -d tests/suite-install -ncc
+  - for f in `ls tests/suite-unit/`; do vendor/bin/atoum -ft -bf tests/bootstrap.php --max-children-number 1 -f tests/suite-unit/$f; done;
```

Comment:
I disagree with the change on the line for suite-unit tests. Using a for loop introduces the risk that Travis will not detect failure for a file in the folder. This is the other line for suite-intrgration which should change to be similar to this one, but we need to fond why the other form fails in Travis and not locally (my computer).

I expect the issue will disappear when we will migrate and rewrite some or most tests in suite-integration